site/content/docs/01-component-format.md

```diff
@@ -189,6 +172,53 @@ Local variables (that do not represent store values) must *not* have a `$` prefi
 </script>
 ```
 
+---
+
+You can create your own stores without relying on [`svelte/store`](docs#svelte_store), by implementing the **store contract**:
+
+1) A store must contain a `.subscribe` method, which must accept as its argument a subscription function. This subscription function must be immediately and synchronously called with the store's current value upon calling `.subscribe`. All of a store's active subscription functions must later be synchronously called whenever the store's value changes.
```

Comment:
Can you change these to either an ordered list with `1. ` etc or an unordered list with `* `? The Markdown parser isn't seeing this as a list, and puts this all in one paragraph.